Dockerfile

```diff
@@ -8,7 +8,8 @@
 FROM alpine:3.16.3
 RUN ["apk", "add", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/edge/community", "skopeo>=1.2.0"]
 
-COPY k8s-image-swapper /
+ARG TARGETARCH
+COPY k8s-image-swapper-$TARGETARCH /k8s-image-swapper
```

Comment:
@darkweaver87 Why is this change needed?

Reply:
Actually it doesn't seem to be needed with go-releaser :-)